### PR TITLE
Feat: Create .github/workflows directory

### LINKED ISSUE #1: Create .github/workflows directory
## Summary
- Create a new .github directory to hold workflows and MD templates in the future
- Inside workflows place a simple yml file(it doesn't required to work or do something for this moment)

### PR DESCRIPTION
**Pull request by lauchaves**
# 📝 Pull Request Title
Create .github/workflows directory

## 🛠️ Issue
- Closes #1 

## 📖 Description
- This pull request adds the necessary directory structure for GitHub Actions to facilitate automation in the project. Additionally, an auto-assign workflow has been implemented to assign the maintainer as a reviewer automatically, and the author of the PR will be added as an assignee.

## ✅ Changes made
- Created a `workflows` subdirectory within `.github`.
- Added an empty YAML file named placeholder.yml to serve as a starting point for future workflows.
- Added an `auto-assign.yml` workflow file that automatically assigns the maintainer as a reviewer and will add the author of the pr as an assignee.

## 📜 Additional Notes
- This setup will allow for the easy addition of workflows later.
- The auto-assign workflow will trigger on opened and reopened pull requests, helping streamline the code review process.
- Further workflows can be added to the workflows directory as needed.

### CODE REVIEW

**Comment by danielcdz** [comment]
@lauchaves that auto-assign workflow is awesome! 😮

File: .github/workflows/auto-assign.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign PR
+        uses: kentaro-m/auto-assign-action@v2.0.0
+        with:
+          reviewers: 'danielcdz'
+          assignees: 'danielcdz'
```
Comment (danielcdz) [inline .github/workflows/auto-assign.yml]: I used to assign the PR to the author. Is it possible to automate this step, taking this into account?

Comment (lauchaves) [reply .github/workflows/auto-assign.yml]: @danielcdz Yes, we can do that! Just one question do you still want to be assigned to the PR, or is it enough to add u as a reviewer only?

Comment (danielcdz) [reply .github/workflows/auto-assign.yml]: Awesome! Just as reviewer is ok 🫡

Comment (lauchaves) [reply .github/workflows/auto-assign.yml]: @danielcdz its updated now!

**Review by danielcdz — APPROVED** [review]
LGTM 🚀

**Pull request merged**